conans/client/cache/cache.py

```diff
@@ -105,14 +105,14 @@ def remove_layout(self, layout):
         layout.remove()
         self._data_cache.remove(ConanReference(layout.reference))
 
+    def get_timestamp(self, ref):
+        return self._data_cache.get_timestamp(ConanReference(ref))
+
     def get_remote(self, ref):
-        return self._data_cache.get_remote(ConanReference(ref))
+        return None
```

Comment:
Mark this as FIXME to be removed? It seems they are here to avoid test breaking?

Reply:
I think it's ok to removed this, I used this in the beginning to develop the feature and forgot to remove that, also the set_remote. I'm removing both.